app/models/event.rb

```diff
@@ -12,6 +12,8 @@
 class Event < ActiveRecord::Base
   translates :title, :introduction, :conclusion
   belongs_to :location
+  # suggestion from Nicholas
+  to_param :title
```

Comment:
I totally missed the `.to_param` class method, very badly tested the behavior in my console :disappointed: 
BUT I was definitely trolling you about `friendly_id` because, you know, I have too much time on my hands :smiling_imp: